netchecker_server/application.py

```diff
@@ -1,19 +1,50 @@
 import datetime
+import logging
+
+try:
+    import http.client as http_client
+except ImportError:
+    # Python 2
+    import httplib as http_client
 
 from flask import abort
 from flask import jsonify
 from flask import Flask
 from flask import request
 import requests
+import signal
+import sys
 
 from netchecker_server import exceptions
 
 
 app = Flask(__name__)
 RECORDS = {}
-
-
 K8S_API_URL = 'http://localhost:8001/api/v1/'
+http_client.HTTPConnection.debuglevel = 1
+
+# You must initialize logging, otherwise you'll not see debug output.
+logging.basicConfig()
```

Comment:
Also I would like to put root logging configuration into separate module and call the set up at entry point level of the application, i.e. in server.py.